style.css

```diff
@@ -0,0 +1,955 @@
+  float: right;
+  font-family: sans-serif;
+}
+
+/* Selector for the grid at the menu bottom container */
+.grid-container-menu-items {
+  margin-top: 16px;
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+}
+
+/* Selector for the containers of each column at the grid at the menu bottom
+  container */
+.menu-item-category {
+  line-height: 10px;
+}
+
+/* Selector for the container of the no header elements inside each column at
+  the grid at the bottom menu container */
+.no-header-menu-item-container {
+  position: relative;
+  bottom: 10px;
+  line-height: 12px;
+  font-family: sans-serif;
+  font-size: 14px;
+}
+
+/* Selector for the headers at the columns at the grid at the bottom menu */
+div.menu-item-category h5 {
+  color: rgb(51, 51, 51);
+  font-size: 13px;
+  font-weight: 900;
+  font-family: sans-serif;
+  height: 25px;
+  line-height: 15px;
+  text-transform: uppercase;
+  width: 150px;
+  margin: 0;
+}
+
+/* Selector for the p elements at the columns at the grid at the bottom menu
+  when the user hoves on it */
+.menu-item-category p:hover {
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+/* Rules for the las column at the grid at the bottom menu */
+.menu-last-column {
+  border-left: 1px solid rgb(226, 226, 226);
+  padding-left: 16px;
+  margin-bottom: 8px;
+}
+
+/* Rules for the second row at the last column at the grid at the bottom menu */
+.menu-last-columm-second-row {
+  border-top: 1px solid rgb(226, 226, 226);
+}
+
+/* Rules for the container of the icon and text at the last column at the grid
+  of the bottom menu */
+.menu-last-columm-icon-text-container {
+  margin-top: 20px;
+}
+
+/* Rules for the container of the icon and text at the last column at the grid
+  of the bottom menu when the user hovers on it */
+.menu-last-columm-icon-text-container:hover {
+  cursor: pointer;
+  text-decoration: underline;
+}
+
+/* Rules for the first container of the icon and text at the last column at the
+  grid of the bottom menu */
+.menu-last-column-icon-text-container-first {
+  margin-top: 12px;
+}
+
+/* Selector for the spans at the last column at the grid at the bottom menu */
+.menu-last-column-link {
+  font-size: 15px;
+  font-weight: bolder;
+  margin-left: 8px;
+  position: relative;
+  bottom: 2px;
+  width: 250px;
+}
+
+/* Rules for the p elements at the last column at the grid of the bottom menu */
+.bottom-menu-last-column-p {
+  display: inline-block;
+  text-transform: capitalize;
+  width: 150px;
+  font-size: 11px;
+  font-weight: 500;
+  text-decoration: none;
+  margin-top: -10px;
+}
+
+/* Rules for the first p element at the last column at the grid of the bottom
+  menu */
+.bottom-menu-last-column-p-first-element {
+  margin-top: 24px;
+}
+
+/* Rules for the first p element at the last column, at the last row at the
+  grid of the bottom menu */
+.mobile-app-p {
+  margin-top: 14px;
+}
+
+/* Rules for the footer tag */
+footer {
+  margin-bottom: 86px;
+  text-align: center;
+}
+
```

Comment:
The footer elements should be arranged with either floats or display flex

Reply:
Hi, the footer elements are now arranged using display: flex 💪